contrib/vcloud-benchmark.py

```diff
@@ -46,12 +48,21 @@ def download_required_jars(config):
         cmd += ["-warn"]
     cmd += ["-retrieve", "lib/vcloud-jars/[artifact](-[classifier]).[ext]"]
 
-    # install vcloud jar and dependencies
-    subprocess.run(
-        cmd,
-        cwd=_ROOT_DIR,
-        shell=vcloudutil.is_windows(),  # noqa: S602
-    )
+    # Provide temporary directory
+    if config.freshIvyCache:
+        temp_dir = tempfile.mkdtemp()
+        cmd += ["-cache", str(temp_dir)]
```

Comment:
Why `str()` here?

Furthermore, it is nicer to indicate the purpose of the directory in case that cleanup does not work, so `prefix` or `suffix` should be set. Otherwise users wonder where these directories come from.